tracer/src/Datadog.Trace/Logging/DirectSubmission/Sink/NullDatadogSink.cs

```diff
@@ -27,5 +27,10 @@ public Task FlushAsync()
         {
```

Comment:
I cannot review the lines above, but shouldn't you have removed the `Dispose` method here as well

Reply:
done in [43e5a69](https://github.com/DataDog/dd-trace-dotnet/pull/2583/commits/43e5a694657ee3a2c735083ba0b2e8e3a33736bd)